contracts/sol6/Dao/KyberStaking.sol

```diff
@@ -288,6 +286,43 @@ contract KyberStaking is IKyberStaking, EpochUtils, ReentrancyGuard {
         return staker;
     }
 
+    /**
+     * @notice return combine data (stake, delegatedStake, delegatedAddress) of a staker
+     * @dev allow to get staker data up to current epoch + 1
+     */
+    function getStakerData(address staker, uint256 epoch)
+        external view
+        returns (
+            uint256 stake,
```

Comment:
here should later update names.
if we find better ones.

Seems to me we should

users will always make every possible mistake
so we should minimize mistake options...
:)